README.md

```diff
@@ -1,35 +1,55 @@
 A fully-functional [bash](https://en.wikipedia.org/wiki/Bash_%28Unix_shell%29) client for [tldr](https://github.com/rprieto/tldr/). This version aims to be the easiest and smallest to set up on a new account, without sacrificing any features.
 
-![tldr screenshot](Screenshot.png?raw=true)
+![tldr screenshot](screenshot.png?raw=true)
 
 # Setup
 
-	mkdir -p ~/bin
-	curl -o ~/bin/tldr https://raw.githubusercontent.com/raylee/tldr/master/tldr
-	chmod +x ~/bin/tldr
+`wget -qO- https://raw.githubusercontent.com/DFabric/DPlatform-ShellCore/master/install.sh | sh`
```

Comment:
I really would like tldr to be the most accessible to the wider range of people.
I agree to not encourage this practice, but please keep (not highlighted if you want) this feature for people (I admit like me) who want to have the things done as efficiently as possible.
This is not as evil (at least here), specially for such an open source script that isn't ran privileged.